collectors/metrics/test/integration/prereq.sh

```diff
@@ -2,50 +2,49 @@
 # Copyright (c) 2021 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project
 
-WORKDIR=`pwd`
+WORKDIR=$(pwd)
 
 setup_kubectl_command() {
-    command -v kubectl	
-    if [ $? -ne 0 ]; then
-		echo "=====Setup kubectl=====" 
-		# kubectl required for kind
-		echo "Install kubectl from openshift mirror (https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.14/openshift-client-mac-4.4.14.tar.gz)" 
-		mv README.md README.md.tmp 
-		if [[ "$(uname)" == "Darwin" ]]; then # then we are on a Mac 
-			curl -LO https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.14/openshift-client-mac-4.4.14.tar.gz 
-			tar xzvf openshift-client-mac-4.4.14.tar.gz  # xzf to quiet logs
-			rm openshift-client-mac-4.4.14.tar.gz
-		elif [[ "$(uname)" == "Linux" ]]; then # we are in travis, building in rhel 
-			curl -LO https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.14/openshift-client-linux-4.4.14.tar.gz
-			tar xzvf openshift-client-linux-4.4.14.tar.gz  # xzf to quiet logs
-			rm openshift-client-linux-4.4.14.tar.gz
-		fi
-		# this package has a binary, so:
+  command -v kubectl
+  if [ $? -ne 0 ]; then
+    echo "=====Setup kubectl====="
```

Comment:
maybe we could take some inspiration from https://github.com/rhobs/configuration/blob/main/Makefile#L216-L218 around handling this? seems cleaner?

Reply:
Yeah, haven't gone through all the shellscripts yet. I might clean them up explicitly in another follow-up!